documentation/tutorial/02-environment.md

```diff
@@ -0,0 +1,80 @@
+# Environment Setup
+
+This is part of the [Charmed Temporal Tutorial](./00-introduction.md). Please
+refer to this page for more information and the overview of the content.
+
+## Minimum Requirements
+
+Before we start, make sure your machine meets the following requirements:
+
+- A laptop or desktop running Ubuntu 22.04 (Jammy) or later. Machines running
```

Comment:
Does the type of the computer matter? I think it might be better to simply say "computer" or an equivalent.